app/src/main/java/com/ciandt/dragonfly/example/features/feedback/FeedbackActivity.kt

```diff
@@ -3,37 +3,119 @@ package com.ciandt.dragonfly.example.features.feedback
 import android.content.Context
 import android.content.Intent
 import android.os.Bundle
+import android.os.PersistableBundle
+import android.support.design.widget.Snackbar
 import com.ciandt.dragonfly.data.model.Model
 import com.ciandt.dragonfly.example.BuildConfig
 import com.ciandt.dragonfly.example.R
+import com.ciandt.dragonfly.example.infrastructure.DragonflyLogger
 import com.ciandt.dragonfly.example.shared.FullScreenActivity
 import com.ciandt.dragonfly.lens.data.DragonflyCameraSnapshot
+import com.ciandt.dragonfly.lens.exception.DragonflyModelException
+import com.ciandt.dragonfly.lens.exception.DragonflyRecognitionException
+import com.ciandt.dragonfly.lens.ui.DragonflyLensFeedbackView
+import com.ciandt.dragonfly.tensorflow.Classifier
 import kotlinx.android.synthetic.main.activity_feedback.*
 
 
 class FeedbackActivity : FullScreenActivity(), FeedbackContract.View {
 
     lateinit private var cameraSnapshot: DragonflyCameraSnapshot
+    lateinit private var model: Model
+
+    var lastRecognizedObjects: ArrayList<Classifier.Recognition>? = null
```

Comment:
We could, but we would need a boolean variable to hold if there are classifications available. I'm still not convinced that aren't cases when Tensorflow doesn't any return any classification...